src/macros.jl

```diff
@@ -511,17 +533,38 @@ julia> using DataFramesMeta
 julia> df = DataFrame(A = 1:3, B = [2, 1, 2]);
 
 julia> @transform df begin
-    a = 2 * :A
-    x = :A .+ :B
-end
+           a = 2 * :A
+           x = :A .+ :B
+       end
+3×4 DataFrame
+ Row │ A      B      a      x
+     │ Int64  Int64  Int64  Int64
+─────┼────────────────────────────
+   1 │     1      2      2      3
+   2 │     2      1      4      3
+   3 │     3      2      6      5
+
+julia> @transform df z = @byrow :A * :B
```

Comment:
do you think it is didactic to show `@transform` without `(` and `)` when you do not use `begin`-`end`?

Reply:
I would generally like this to be the default way of writing in the future. I think it's cleaner and more declarative. So maybe I can have a future PR change some of these en-masse if people agree with me.